src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/SerializableOpticalDuplicatesFinder.java

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.hellbender.utils.read.markduplicates;
+
+import picard.sam.markduplicates.util.OpticalDuplicateFinder;
+import java.io.Serializable;
+
+/**
+ * An intermediate class flagged as being serializable so that the Picard OpticalDuplicateFinder can be serialized despite
+ * it not being marked as such. There is not state outside of construction at the time of creating this pr.
```

Comment:
the second sentence is more confusing than helpful

Reply:
done